index.js

```diff
@@ -13,7 +15,8 @@ const replacer = (match, group) => {
 };
 
 class WhiteOut extends Stream.Transform {
-  constructor (rules, options) {
+  constructor (rules, root, options) {
+    Assert.ok(root == null || typeof root === 'string', 'root must be a string'); // eslint-disable-line eqeqeq
```

Comment:
I guess that makes sense. Although it would make more sense to only support a single `options` object.